### PR TITLE
Add recipe for the Python package quadprog

### PR DESCRIPTION
**Pull request by nfahlgren**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/quadprog/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  host:
+    - cython
+    - pip
+    - python
+  run:
+    - cython
+    - python
+    - numpy
+
+test:
```
Comment (marcelotrevisani) [inline recipes/quadprog/meta.yaml:28]:
```suggestion
test:
  imports:
    - quadprog
```

File: recipes/quadprog/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  commands:
+    - python -c "from quadprog import solve_qp"
```
Comment (marcelotrevisani) [inline recipes/quadprog/meta.yaml:30]:
```suggestion
    - pip check
  requires:
    - pip
```

File: recipes/quadprog/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+about:
+  home: "https://github.com/rmcgibbo/quadprog"
+  license: GPL-2.0-or-later
+  license_family: GPL2
+  license_file: LICENSE
+  summary: "Quadratic Programming Solver"
+  doc_url: 
+  dev_url:
```
Comment (marcelotrevisani) [inline recipes/quadprog/meta.yaml:39]:
```suggestion
```

File: recipes/quadprog/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - cython
+    - pip
+    - python
+  run:
+    - cython
+    - python
+    - numpy
```
Comment (marcelotrevisani) [inline recipes/quadprog/meta.yaml:26]: could you please double-check this dependency here ``numpy``? if the project needs it

Comment (nfahlgren) [reply recipes/quadprog/meta.yaml:26]: Thank you for the suggestions @marcelotrevisani! I can confirm `numpy` is a dependency

**Pull request merged**